### PR TITLE
Improve default suggestion in workflow section for Test Operator

### PR DESCRIPTION
**Pull request by jakubskunda**
When someone wants to quickly work with this section an error occured. In this patch we change values in stepName that it would start with lowercase letter.

### CODE REVIEW

**Review by kopecmartin — APPROVED** [review]
lgtm

**Review by lpiwowar — APPROVED** [review]
LGTM! :+1: 

Thank you! Let's wait for the job result before you merge this.

**Pull request merged**
